packages/core/src/events/handlers/createEvaluationResultJob.ts

```diff
@@ -25,10 +31,32 @@ export const createEvaluationResultJob = async ({
   const providerLog = await unsafelyFindProviderLogByUuid(providerLogUuid)
   if (!providerLog) throw new NotFoundError('Provider log not found')
 
-  await createEvaluationResult({
+  const evaluationResult = await createEvaluationResult({
     evaluation,
     documentLog,
     providerLog,
     result: (response as ChainObjectResponse).object,
   }).then((r) => r.unwrap())
+  evaluationResult.id
+
+  const { evaluationResultsScope, baseQuery } = createEvaluationResultQuery(
+    evaluation.workspaceId,
+    database,
+  )
+  const result = await baseQuery
+    .where(eq(evaluationResultsScope.id, evaluationResult.id))
+    .limit(1)
```

Comment:
hmm weird way of building the query why didn't u use a repository?

Reply:
`baseQuery` is using repositories

Reply:
but why not `EvaluationResultsRepository(...).find(evaluationResult.id)` y ya esta?

Reply:
The baseQuery brings same aggregations used in the table